R/Table.R

```diff
@@ -50,6 +50,15 @@ odbc_write_table <-
       dbExecute(conn, sql)
     }
 
+    fieldDetails <-
+      tryCatch({
+        details <- odbcConnectionColumns(conn, name)
+        datails <- details[match(names(values), details$column_name)]
+        details[, c("ordinal_position", "data_type", "column_size", "decimal_digits")]
```

Comment:
This call has no effect, did you mean to assign it back to details?

Reply:
Jim - the output of the `tryCatch` block is assigned to `fieldDetails` -  the line you quoted there is what gets assigned in the success code-path.

Let me know if you think there is a better approach.

Reply:
It was due to the indentation style used, I would normally put the `tryCatch()` line on the same line as the assignment.

Reply:
Will update - thanks.